SherpaFrontEnd.Tests/Acceptance/DownloadResponsesAcceptanceTest.cs

```diff
@@ -0,0 +1,165 @@
+        _httpClientFactory.Setup(factory => factory.CreateClient("SherpaBackEnd")).Returns(httpClient);
+        var auth = _testCtx.AddTestAuthorization();
+        auth.SetAuthorized("Demo user");
+        auth.SetClaims(new[] { new Claim("username", "Demo user") });
+        _surveyService = new SurveyService(_httpClientFactory.Object);
+        _testCtx.Services.AddSingleton<ISurveyService>(_surveyService);
+        _teamDataService = new TeamServiceHttpClient(_httpClientFactory.Object);
+        _testCtx.Services.AddSingleton<ITeamDataService>(_teamDataService);
+
+        _navManager = _testCtx.Services.GetRequiredService<FakeNavigationManager>();
+    }
+
+    [Fact]
+    public async Task UserShouldBeAbleToClickOnDownloadResponsesButtonAndSendDownloadRequest()
+    {
+        var teamId = Guid.NewGuid();
+        var team = new Team();
+        var teamJson = await JsonContent.Create(team).ReadAsStringAsync();
+        var teamResponse = new HttpResponseMessage()
+        {
+            StatusCode = HttpStatusCode.OK,
+            Content = new StringContent(teamJson)
+        };
+        _handlerMock.SetupRequest(HttpMethod.Get, $"/team/{teamId}", teamResponse);
+
+        var surveyId = Guid.NewGuid();
+        var surveyTitle = "title";
+        var surveys = new List<Survey>()
+        {
+            new Survey(surveyId, new User(Guid.NewGuid(), "Lucia"), Status.Draft, new DateTime(), surveyTitle,
+                "Description", Array.Empty<Response>(), null, new Template("Hackman"))
+        };
+        var surveyJson = await JsonContent.Create(surveys).ReadAsStringAsync();
+        var surveyResponse = new HttpResponseMessage()
+        {
+            StatusCode = HttpStatusCode.OK,
+            Content = new StringContent(surveyJson)
+        };
+        _handlerMock.SetupRequest(HttpMethod.Get, $"/team/{teamId}/surveys", surveyResponse);
+
+        var downloadSurveyPath = $"survey/{surveyId}/responses";
+        var csvText = "Responses,1,2,3";
+        var fileBytes = Encoding.UTF8.GetBytes(csvText);
+        var downloadSurveyResponse = new HttpResponseMessage()
+        {
+            StatusCode = HttpStatusCode.OK,
+            Content = new ByteArrayContent(fileBytes)
+        };
+        _handlerMock.SetupRequest(HttpMethod.Get, downloadSurveyPath, downloadSurveyResponse);
+
+        // Given an Org Coach is on the survey tab inside a team
+        var appComponent = _testCtx.RenderComponent<App>();
+        var teamContentPageSurveys = $"/team-content/{teamId}/surveys";
+        _navManager.NavigateTo(teamContentPageSurveys);
+
+        appComponent.WaitForAssertion(() =>
+            Assert.Equal($"http://localhost/team-content/{teamId.ToString()}/surveys", _navManager.Uri));
+
+        // When they click on the 3 dots next to a survey (this can't be clicked from the test)
+        var downloadButton =
+            appComponent.FindElementByCssSelectorAndTextContent("button", "Download responses in .xlsx");
```

Comment:
Can we change the text of this button to just "Download responses file" or "Download responses in .csv"?